# Data Science Tools and Ecosystem

In this notebook, Data Science Tools and Ecosystem are summarized.

***Objectives:***
- Python
- Java
- C++
- Scala
- Julia

Some of the popular languages that Data Scientists use are:
1. Python
2. C++
3. Java
4. Scala
5. Julia

Some of the commonly used libraries used by Data Scientists include:
1. Pandas
2. Numpy
3. Scikit
4. Keras
5. Pytorch

|Number|Data Science Tools|
|------|--------------|
|1. | Spyder |
|2.| Jypiter |
|3. | RStudio | 

### Below are a few examples of evaluating arithmetic expressions in Python


This a simple arithmetic expression to multiply then add integers:


In [5]:
(3*4)+5

17

This will convert 200 minutes to hours by dividing by 60:

In [22]:
200/60

3.3333333333333335

Thus, the obtained time is 3.33 hours

## Author
Đorđe Antić